### DISCUSSION TITLE
Access.log - Switch the default?

### DISCUSSION THREAD

**Discussion by cescoffier** [quarkusio/quarkus#43587] [Design Discussions]
At the moment, there are three approaches to writing the HTTP server log:

- built-in
- JBoss Logging
- Vert.x

The built-in approach offers few features (See https://github.com/quarkusio/quarkus/discussions/43413).
The JBoss logging one has more features.
The Vert.x on can only be enabled programmatically.

We should consider using the JBoss Logging one as default and document the migration (we would not remove the built-in one, just switch the default).

WDYT?

**Comment by cescoffier** [comment]
\CC  @gsmet @geoand @dmlloyd @maxandersen

**Comment by geoand** [comment]
+1 for JBoss Logging if possible. IIRC, there was some issue asking for something like that.

**Reply by geoand** [reply]
It likely would also help with things like https://github.com/quarkusio/quarkus/issues/43570
